config/rbac/role.yaml

```
@@ -62,3 +62,16 @@ rules:
   - get
   - list
   - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: manager-role
+  namespace: catalogd-system
```

```suggestion
  namespace: system
```
Doesn't kustomize add the `catalogd` prefix?

---

I think so. This was autogenerated based on the rbac markers in the controller though. I guess I can update the namespace there to be "system" instead. FWIW this does work as expected so not sure if kustomize is smart enough to recognize that the "catalogd-" prefix is already added or not.